CHANGELOG.rst

```diff
@@ -14,10 +14,11 @@ Change Log
 Unreleased
 ----------
 
-[0.71.0] - 2018-07-13
+[0.71.0] - 2018-07-20
```

Comment:
@irfanuddinahmad You forgot to update the semantic versioning  `[0.71.1]`.

Also we also need to update this version in https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py#L7

For reference please view this PR by @mushtaqak https://github.com/edx/edx-enterprise/pull/347/files

I suggest you to create a new PR with these updates before publishing a new `edx-enterprise` release.

Reply:
@irfanuddinahmad its clear now that we tagged along @saleem-latif PR https://github.com/edx/edx-enterprise/pull/344/files for releasing your change https://github.com/edx/edx-enterprise/pull/348. 🆒